### DISCUSSION TITLE
Cards: Empire of war

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#20] [Ideas]
Almost missed this non-trivial condition. Suggestion:
> **Performed as an Event by the US player**: US conducts a Destabilization Ops with this cards Ops value in a Disputed country. If successful, US repeats this in a non-Disputed country.

I think this should correctly cover all cases; played by an East player then performed by US or EU, played by US directly, or played by EU.

**Comment by rodrigoSantamaria** [comment]
Do you think it's necessary the "Performed as an Event by the US player" clause? If the text says "US conducts...", i think it's clear it's not UE, which would be the other possible player.

**Reply by RebelusQuo** [reply]
The original text is:
> If you are the US player, make a coup with this ops value on a conflictive country. If succesful, make a second coup with the same ops value on a non-conflictive country.

I interpreted this as a no-op for everyone except the US player, not that US get to do it regardless of who performs the Event. Was it meant to work like the later?

**Comment by rodrigoSantamaria** [comment]
Yes, it means that if EU uses it for the texts, it does nothing. Russia/China will never use it for the text, as is it a W card.

**Reply by RebelusQuo** [reply]
But if an East player plays the card, then the Event happens and it will be performed by the next West player by Header Order. if that next player is US then they get to benefit, but if it's EU nothing happens. Correct?

I take the wording "[power] [does something]" to means that 1. it will always happen if the Event happens, and 2. The indicated power will get to decide about the effect regardless of who performs the Event.

Basically if the following is true:
Played by US as an Event: US performs the Event effect.
Played by EU as an Event: the Event effect has no effect.
Played by East player, US is next by Header Order: US performs the Event effect.
Played by East player, EU is next by Header Order: the Event effect has no effect.

then the specially worded condition is needed. if the effect starts with "[power] [does something]" then the matrix would be:
Played by West player as an Event: US performs the Event effect.
Played by East player: US performs the Event effect.

**Reply by rodrigoSantamaria** [reply]
> Basically if the following is true:
>Played by US as an Event: US performs the Event effect.
>Played by EU as an Event: the Event effect has no effect.
>Played by East player, US is next by Header Order: US performs the Event effect.
>Played by East player, EU is next by Header Order: the Event effect has no effect.
This is correct.

**Reply by RebelusQuo** [reply]
Acknowledged.
